docs/Installing.asciidoc

```diff
@@ -308,12 +308,14 @@ For a local instance setup you can simply execute the script:
 This will automatically setup a local apache http proxy. Read on for more
 detailed setup instructions with all the details.
 
+It is required to run openQA behind an http proxy (apache, nginx, etc..). See the
+*openqa.conf.template* config file in */etc/apache2/vhosts.d* (openSUSE) or
+`/etc/httpd/conf.d` (Fedora) when using apache2 or the config file in
+`/etc/nginx/conf.d` for nginx.
```

Comment:
you supplied it in vhosts.d though for nginx altough I am not sure this is correct.